From f3fcf09cee87493665e8915992c0e735b1ad0176 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Luk=C3=A1=C5=A1=20Karas?= <lukas.karas@centrum.cz>
Date: Sat, 14 Nov 2020 20:20:09 +0100
Subject: [PATCH 3/3] check that port is in valid range

Signed-off-by: Lukas Karas <lukas.karas@centrum.cz>
---
 http.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/http.c b/http.c
index 72db08fb..be54fb2e 100644
--- a/http.c
+++ b/http.c
@@ -764,6 +764,8 @@ int internal_parse_url(const char *url, char **res_proto, char **res_host,
 		if (!*end) {
 			*port_str = 0;
 			port = new_port;
+			if (port <= 0 || port > 0xffff)
+				return -EINVAL;
 		}
 	}
 
-- 
2.25.1

